HW0/P6/P6B.py

```diff
@@ -11,21 +12,27 @@ def burnTime(t):
     N = 16  # The number of jobs
     P = 4   # The number of processes
 
-    # A thread pool of P processes
-    pool = mp.Pool(P)
+    pool = mp.Pool(P) #create a pool of P processes
 
     # Use a variety of wait times
-    ratio = []
-    wait_time = []
-
+    ratio = [] #empty ratio array to be filled in below
+    #wait_time = np.array([10**-6,10**-5,10**-4,0.5*10**-2,0.5*10**-1,10**-1,1])
+    wait_time = np.array([10**-6,10**-5,10**-4,10**-3,10**-2,10**-1,1])
     for t in wait_time:
         # Compute jobs serially and in parallel
-        # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
-
+        begin = time.time()
+        pool.map(burnTime(t),range(N))                        
```

Comment:
No problem Andrew. The [1]*16 is just means [1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]. Then map will pass in each value in the array(in this case will pass 1 each time) to burnTime. Thus executing burnTime(1) 16 times